core/src/test/java/org/testcontainers/junit/ParameterizedDockerfileContainerTest.java

```diff
@@ -22,30 +22,30 @@ public class ParameterizedDockerfileContainerTest {
     public GenericContainer container;
 
     public ParameterizedDockerfileContainerTest(String baseImage, String expectedVersion) {
-        container = new GenericContainer(new ImageFromDockerfile().withDockerfileFromBuilder(builder -> {
-                builder
-                        .from(baseImage)
-                        // Could potentially customise the image here, e.g. adding files, running
-                        //  commands, etc.
-                        .build();
-            })).withCommand("top");
+        container =
+            new GenericContainer(
+                new ImageFromDockerfile()
+                    .withDockerfileFromBuilder(builder -> {
+                        builder
+                            .from(baseImage)
+                            // Could potentially customise the image here, e.g. adding files, running
+                            //  commands, etc.
+                            .build();
+                    })
+            )
+                .withCommand("top");
         this.expectedVersion = expectedVersion;
     }
 
     @Parameterized.Parameters(name = "{0}")
     public static Object[][] data() {
-        return new Object[][] {
-                { "alpine:3.12", "3.12"},
-                { "alpine:3.13", "3.13"},
-                { "alpine:3.14", "3.14"}
-        };
+        return new Object[][] { { "alpine:3.12", "3.12" }, { "alpine:3.13", "3.13" }, { "alpine:3.14", "3.14" } };
```

Comment:
WDYT about adding `//` after `new Object[][] {` to force Prettier break it line-by-line?